types/time.go

```diff
@@ -1765,7 +1771,7 @@ func ParseDuration(sc *stmtctx.StatementContext, str string, fsp int8) (Duration
 		return ZeroDuration, ErrTruncatedWrongVal.GenWithStackByArgs("time", str)
 	}
 
-	return d.RoundFrac(fsp)
+	return d.RoundFrac(fsp, sc.TimeZone)
```

Comment:
`sc` may be nil for some test case

Reply:
So I think it's better to fix `nil` for the test cases.

Reply:
OK, I think I need to fix the test.